src/app/games/pente/PenteRules.ts

```diff
@@ -93,13 +100,13 @@ export class PenteRules extends ConfigurableRules<PenteMove, PenteState, GobanCo
         return captures;
     }
 
-    public getGameStatus(node: PenteNode): GameStatus {
+    public getGameStatus(node: PenteNode, config: MGPOptional<PenteConfig>): GameStatus {
         const state: PenteState = node.gameState;
         const opponent: Player = state.getCurrentOpponent();
-        if (10 <= state.captures.get(opponent)) {
+        if (config.get().winAfterNCapture * 2 <= state.captures.get(opponent)) {
```

Comment:
ok c'est donc paires de pièces pour gagner, qui de dire "number of pieces captured to win"? Ainsi on évite toute ambigüité pour les débilos comme moi